site/pages/testData/mockData.ts

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
I'd move this testData folder outside of the pages directory

Reply:
done